### PR TITLE
Add changelog and tag v1.0.0

### PR DESCRIPTION
**Pull request by frm**
The project seems mature enough to mark it as `v1.0.0` but I'd actually prefer feedback on this.

I took the liberty of using semver and adding the [Keep A Changelog](http://keepachangelog.com/en/0.3.0/) format changelog.

After merging, I'll create the release draft and we can take it from there.

### CODE REVIEW

**Comment by frm** [comment]
It also fixes #121

**Comment by desyncr** [comment]
I also believe `1.0.0` is a good starting point. Project is quite stable right now and won't face huge changes in the short term.

**Review by desyncr — APPROVED** [review]
I haven't used `Unreleased` section before but I have no issues against it, seems reasonable.

**Comment by desyncr** [comment]
After @jedahan OK I think we should merge it and release right away. No need to wait for pending issues/PR.

What you think @frmendes?

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/)
+and this project adheres to [Semantic Versioning](http://semver.org/).
+
+## [Unreleased]
+### Added
+
+### Changed
+
+## 1.0.0 - 2017-04-05
```
Comment (jedahan) [inline CHANGELOG.md:12]: maybe we want to link to the current revision here?

**Review by jedahan — CHANGES_REQUESTED** [review]
Good context, just want to add one link then we can push

File: CHANGELOG.md (hunk 1 above)
Comment (jedahan) [reply CHANGELOG.md:12]: Or is that impossible because we have not pushed it yet?

Comment (frm) [reply CHANGELOG.md:12]: We have not pushed it yet. I could probably link to the revision and guess the correct URL. However, [Keep A Changelog](http://keepachangelog.com/en/0.3.0/) suggests linking the comparison between 2 consecutive versions.

**Comment by frm** [comment]
@desyncr 

> I haven't used `Unreleased` section before but I have no issues against it, seems reasonable.

I haven't either but I'm willing to give it a shot and try that format. If you have any suggestions I'd love to hear them!

> I think we should merge it and release right away. No need to wait for pending issues/PR.

I agree 😄 as soon as @jedahan approves it, I'm releasing. Should I just make a release draft or can I release it right away as "Initial release"?

**Review by jedahan — APPROVED** [review]

**Comment by jedahan** [comment]
Initial release is good. I wonder if we should do a little coordinated marketing around the release. Posting screenshots on r/unixporn, twitter, etc...

**Comment by frm** [comment]
I agree with the marketing. Do you want to talk about this in an issue or gitter? @jedahan @desyncr

**Comment by desyncr** [comment]
We should probably hang out on gitter to talk about it (there will be too much back and forward). We should set a date/time we can all hang out a couple of minutes.

**Comment by frm** [comment]
@desyncr [Sure, sounds great!](https://youtu.be/GLbzN1Q1Agw?t=34s)

cc @jedahan

**Comment by jedahan** [comment]
I am actually on vacation out of my normal country so it is a bit difficult for me to communicate but I will try. On April 19 I will be more reliable.

**Pull request merged**